### PR TITLE
RGI_MAIN: Replace multiple ifs with a for loop

### PR DESCRIPTION
**Pull request by amizeranschi**
The loop checks if files exist before moving them to temp/

## PR checklist

Closes #376

### CODE REVIEW

**Comment by amizeranschi** [comment]
@jfy133

**Review by jfy133 — APPROVED** [review]

**Comment by jfy133** [comment]
Thanks @amizeranschi ! Please feel free to consider joining the nf-core github org (instructions [here](https://nf-co.re/join)), that way you don't have to wait for approval for tests to run next time :D

**Pull request merged**
